foglamp.version

```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```

Comment:
@Singhal-Vaibhav Missing service_notification.version. Please add.

Here we have foglamp.version as well, is this really required @pintomax @MarkRiddoch
I think we are NOT keeping foglamp.version file in rule- plugins; as direct dependency is notification service which has dependency check for foglamp version

Reply:
We need foglamp.version  to be updated for any plugin.
For notification plugins we something like this:

Depends: foglamp (>=1.7),foglamp-service-notification (>=1.7.0)

>=1.7 comes from "foglamp.version and" ... notification (>=1.7.0) comes from "service_notification.version"